### PR TITLE
blui 4598 update js docs

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-4598.

#### Changes proposed in this Pull Request:

- update components & screen JS Docs
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

- verify verbiage on props
- verify verbiage on js doc descriptions
- verify all props are covered

### CODE REVIEW

File: login-workflow/src/components/WorkflowCard/WorkflowCardInstructions.tsx [hunk 1]
```diff
@@ -3,6 +3,15 @@ import Divider from '@mui/material/Divider';
 import Typography from '@mui/material/Typography';
 import { WorkflowCardInstructionProps } from './WorkflowCard.types';
 
+/**
+ * Component that renders the instructions content for the workflow card.
+ *
+ * @param instructions
+ * @param divider
```
Comment (ektaghag-eaton) [inline login-workflow/src/components/WorkflowCard/WorkflowCardInstructions.tsx:10]: Do we need to add any description here?

File: login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx [hunk 2]
```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import CardContent, { CardContentProps } from '@mui/material/CardContent';
 
+/**
+ * Component that renders the body content for the workflow card.
+ *
+ * @param children
```
Comment (ektaghag-eaton) [inline login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx:7]: Description needed ?

File: login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+ * @param otherCardContentProps
```
Comment (daileytj) [inline login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx:8]: no need for `otherCardContentProps` to be here

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 4]
```diff
@@ -9,12 +9,15 @@ import { useAuthContext } from '../../contexts';
...
  *
- * @param title to display in the header
- * @param icon to display in the icon
- * @param description to display the contact support content
+ * @param title the title for the screen
+ * @param icon the icon to display in the header
+ * @param emailSupportTitle to display the contact support title
+ * @param emailSupportContent to display the contact support content
+ * @param phoneSupportTitle to display the phone support title
+ * @param phoneSupportContent to display the phone support content
```
Comment (daileytj) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:17]: Can we make all of these complete phrases? e.g.,
```suggestion
 * @param title the title for the screen
 * @param icon the icon to display in the header
 * @param emailSupportTitle text to display as the contact support title
 * @param emailSupportContent content to display as the contact support content
 * @param phoneSupportTitle text to display as the phone support title
 * @param phoneSupportContent content to display as the phone support content
```

File: login-workflow/src/screens/CreateAccountScreen/CreateAccountScreenBase.tsx [hunk 5]
```diff
@@ -12,10 +12,11 @@ import ErrorManager from '../../components/Error/ErrorManager';
  * Component that renders a screen for the user to enter their email address to start the
  * account creation process.
  *
- * @param emailLabel label for the textfield
- * @param initialValue used to pre-populate the email input field
+ * @param emailLabel label for the email field
+ * @param initialValue initial value for the email text field
  * @param emailValidator used to test the input for valid formatting
```
Comment (daileytj) [inline login-workflow/src/screens/CreateAccountScreen/CreateAccountScreenBase.tsx:17]:
```suggestion
 * @param emailValidator function used to test the input for valid formatting
```

File: login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreen.tsx [hunk 6]
```diff
@@ -10,6 +10,21 @@ import CheckCircle from '@mui/icons-material/CheckCircle';
 import { LinkStyles } from '../../styles';
 import { useErrorManager } from '../../contexts/ErrorContext/useErrorManager';
 
+/**
+ * Component renders a screen with forgot password for support with the application.
+ *
+ * @param emailLabel label for the email field
+ * @param initialEmailValue initial value for the email text field
+ * @param emailValidator used to test the input for valid format
```
Comment (daileytj) [inline login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreen.tsx:18]:
```suggestion
 * @param emailValidator function used to test the input for valid formatting
```

File: login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx [hunk 7]
```diff
@@ -15,6 +15,21 @@ type ForgotPasswordScreenBaseProps = Omit<ForgotPasswordScreenProps, 'slots'> &
     slots: { SuccessScreen: (props: SuccessScreenProps) => JSX.Element };
 };
 
+/**
+ * Component renders a screen with forgot password for support with the application.
+ *
+ * @param emailLabel label for the email field
+ * @param initialEmailValue initial value for the email text field
+ * @param emailValidator used to test the input for valid format
```
Comment (daileytj) [inline login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx:23]:
```suggestion
 * @param emailValidator function used to test the input for valid formatting
```

File: login-workflow/src/components/WorkflowCard/WorkflowCardInstructions.tsx (hunk 1 above)
Comment (daileytj) [inline login-workflow/src/components/WorkflowCard/WorkflowCardInstructions.tsx:10]:
```suggestion
 * @param instructions text to display as instructions
 * @param divider whether or not to show a divider below the instructions
```

File: login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx (hunk 2 above)
Comment (JeffGreiner-eaton) [reply login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx:7]: not sure, is description a prop type for this or is it from somewhere else

Comment (daileytj) [reply login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx:7]: I think @ektaghag-eaton was questioning "Does the `children` prop need a description?"

Comment (daileytj) [reply login-workflow/src/components/WorkflowCard/WorkflowCardBody.tsx:7]:
```suggestion
 * @param children content to render in the WorkflowCardBody
```

File: login-workflow/src/screens/LoginScreen/LoginScreen.tsx [hunk 8]
```diff
@@ -38,6 +38,10 @@ const EMAIL_REGEX = /^[A-Z0-9._%+'-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
  * @param projectImage image to display at the top of the screen
  * @param header header to display at the top of the screen
  * @param footer footer to display at the bottom of the screen
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ * @param WorkflowCardActionsProps props that will be passed to the WorkflowCardActions component
```
Comment (daileytj) [inline login-workflow/src/screens/LoginScreen/LoginScreen.tsx:44]: Login doesn't have these 3 props

File: login-workflow/src/screens/LoginScreen/LoginScreenBase.tsx [hunk 9]
```diff
@@ -47,6 +47,10 @@ import { LinkStyles } from '../../styles';
  * @param projectImage image to display at the top of the screen
  * @param header header to display at the top of the screen
  * @param footer footer to display at the bottom of the screen
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ * @param WorkflowCardActionsProps props that will be passed to the WorkflowCardActions component
```
Comment (daileytj) [inline login-workflow/src/screens/LoginScreen/LoginScreenBase.tsx:53]: Login doesn't have these 3 props

**Review by daileytj — APPROVED** [review]

**Pull request merged**